doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -923,14 +923,8 @@ dt.label {
   padding-right: 0.5rem;
 }
 
-/* copy button */
-div.highlight:hover span.copybutton {
-  background-color: #3F556B;
-  color: white;
-}
-
-div.highlight:hover span.copybutton:hover {
-    background-color: #20252B;
+button.copybtn {
+  border: 0!important;
```

Comment:
Do you think there is a way to do this without having to use `!important`? Asking because one of the few things I know about CSS is that trying to avoid using `!important` is a good thing/you should only use it when you have to.

Reply:
You are right, it can be done without. I removed the property.